### PR TITLE
[ADD] Weekly Reports #427

### PR DESCRIPTION
**Pull request by eilst**

### CODE REVIEW

File: project_resource_calendar/views/equipment_calendar_view.xml [hunk 1]
```diff
@@ -88,9 +88,9 @@
         <field name="arch" type="xml">
            <xpath expr="//field[@name='message_needaction']" position="after">
                 <field name="state"/>
+                <field name="weekday"/>
```
Comment (rimbendhaou) [inline project_resource_calendar/views/equipment_calendar_view.xml:131]: we need to display this?

File: project_resource_calendar/report/calendar_event_report_view.xml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <report id="project_resource_calendar_report"
```
Comment (rimbendhaou) [inline project_resource_calendar/report/calendar_event_report_view.xml:3]: project_resource_calendar_weekly_event_report

**Review by rimbendhaou — CHANGES_REQUESTED** [review]
the name of the room should appear only in the top (title) not in each line (according to @wbeverly ).
Better to reduce to margin size of the report

File: project_resource_calendar/views/equipment_calendar_view.xml (hunk 1 above)
Comment (eilst) [reply project_resource_calendar/views/equipment_calendar_view.xml]: Yes

File: project_resource_calendar/report/calendar_event_report_view.xml (hunk 2 above)
Comment (rimbendhaou) [reply project_resource_calendar/report/calendar_event_report_view.xml:3]: this is not a view better to rename the file to (calendar_event_weekly_report.xml)

File: project_resource_calendar/__manifest__.py [hunk 3]
```diff
@@ -26,6 +26,9 @@
         'views/resource_view.xml',
         'views/room_view.xml',
         'views/instrument_view.xml',
+        'templates/assets.xml',
+        'report/calendar_event_report_view.xml',
```
Comment (rimbendhaou) [inline project_resource_calendar/__manifest__.py:30]: Better to rename the file to calendar_event_weekly_report we will have also another report with this name in another PR

**Review by rimbendhaou — CHANGES_REQUESTED** [review]

**Pull request merged**